src/test/resources/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/validering_utgifter.feature

```diff
@@ -0,0 +1,75 @@
+# language: no
+# encoding: UTF-8
+
+Egenskap: Beregning barnetilsyn - validering av utgifter
+
+  Scenario: Sender inn tomme utgifter
+
+    Gitt følgende støndsperioder
+      | Fom     | Tom     |
+      | 01.2023 | 02.2023 |
+
+    Gitt følgende utgifter for barn: 1
+      | Fom | Tom | Utgift |
+
+    Når beregner
+
+    Så forvent følgende feil: Utgiftsperioder mangler
+
+  Scenario: Sender inn usorterte utgifter
+
+    Gitt følgende støndsperioder
+      | Fom     | Tom     |
+      | 01.2023 | 02.2023 |
+
+    Gitt følgende utgifter for barn: 1
+      | Fom     | Tom     | Utgift |
+      | 02.2023 | 02.2023 | 100    |
+      | 01.2023 | 01.2023 | 100    |
+
+    Når beregner
+
+    Så forvent følgende feil: Utgiftsperioder er ikke sortert
+
+  Scenario: Sender inn overlappende utgiftsperioder
+
+    Gitt følgende støndsperioder
+      | Fom     | Tom     |
+      | 01.2023 | 03.2023 |
+
+    Gitt følgende utgifter for barn: 1
+      | Fom     | Tom     | Utgift |
+      | 01.2023 | 02.2023 | 100    |
+      | 02.2023 | 01.2023 | 100    |
+
+    Når beregner
+
+    Så forvent følgende feil: Utgiftsperioder overlapper
+
+  Scenario: Sender inn 0 som utgift
+
+    Gitt følgende støndsperioder
+      | Fom     | Tom     |
+      | 01.2023 | 03.2023 |
+
+    Gitt følgende utgifter for barn: 1
+      | Fom     | Tom     | Utgift |
+      | 01.2023 | 02.2023 | 0      |
+
+    Når beregner
+
+    Så forvent følgende feil: Utgiftsperioder inneholder ugyldig verdi: 0
```

Comment:
Burde ikke 0 være en gyldig verdi? Kan se for meg at noen sb kanskje skriver inn 0 for å stoppe en utgift eller hvis et av barna bare har utgift i en del av perioden og det andre barnet har hele perioden. F.eks.:

Barn 1: 
Januar: 0kr
Februar-juni: 1000kr

Barn 2: 
Januar-juni: 1000kr

Burde ikke være lov å kun legge inn 1 utgift på 0kr kanskje, men hva gjør SB dersom bruker søker for 2 barn og det viser seg å kun være utgifter på 1 barn? De har ikke da noen mulighet til å fjerne dem 🥲

Reply:
Kanskje ikke, det beror på hvordan vi løser det